avalanche/training/supervised/er_ace.py

```diff
@@ -47,6 +49,7 @@ def __init__(
         ] = default_evaluator,
         eval_every=-1,
         peval_mode="epoch",
+        **kwargs
```

Comment:
Why is **kwargs needed also for top-level classes? There shouldn't be any extra arguments. I'm worried because unused kwargs would go unnoticed with this solution.
My understanding is that we need it for classes that are not top-level so that the constructor works for any MRO.

Reply:
Yes, makes sense. But consider that it's still a WIP and I still need to work on this PR!